superset-frontend/src/explore/components/DataTablesPane/index.tsx

```diff
@@ -242,9 +255,13 @@ export const DataTablesPane = ({
         ...prevIsLoading,
         [resultType]: true,
       }));
+
+      chartUpdateCacheInvalidationStatus(false);
+
       return getChartDataRequest({
         formData: queryFormData,
         resultFormat: 'json',
+        force: !!latestQueryCacheInvalidatedRef.current,
         resultType,
         ownState,
       })
```

Comment:
I remember that the `force` field is in the explore state, but I can not get a `True` value even though I clicked the `cached` button. Could we resolve this issue through change the `force` value?

![image](https://user-images.githubusercontent.com/2016594/161919457-d7aac73c-c51f-4c15-a843-395296291c26.png)

Reply:
As far as I can tell, that force is used for something else, which is tracked in the `ExploreViewContainer`.
I also cannot find where in the `exploreReducer` that is being modified, but updating that value there will have other consequences

Reply:
@diegomedina248 I pushed a [branch](https://github.com/zhaoyongjie/incubator-superset/commit/ffdba0532977ca1663f27a2aa012fed7c82220f8) in my repository to fix this issue. discussion welcome.

Reply:
@zhaoyongjie looks good to me! feel free to close this PR once yours lands here. Thanks

Reply:
@zhaoyongjie were you indeed opening a PR to resolve this, or just providing some advice/examples?

Reply:
@rusackas @diegomedina248 I will open a new PR today. Thanks.

Reply:
opened PR at https://github.com/apache/superset/pull/19932.